python/EnumWrapper.cpp

```diff
@@ -72,16 +72,16 @@ namespace FreeOrionPython {
             .value("stockpile",         BT_STOCKPILE)
         ;
         enum_<ResourceType>("resourceType")
-            .value("industry",      RE_INDUSTRY)
-            .value("trade",         RE_TRADE)
-            .value("research",      RE_RESEARCH)
-            .value("stockpile",     RE_STOCKPILE)
+            .value("industry",          RE_INDUSTRY)
```

Comment:
Random indentation change, only bloats the diff and provokes rebase conflicts.

Reply:
not random; now it lines up with the surrounding code

(pointless you can argue, which I'll disagree with, but it's not random)

Reply:
Yes it is, it marks lines as relevant change, when in fact there is no relevant change to the logic.  If you want to help people reviewing and spotting errors, you don't throw in changes that are unrelated to the code to review.  There is a a certain threshold of a person can ingest and understand in a single read. If you exceed that the review will become sloppy and the reviewer will miss changes.  As I already said in the final comment I didn't bother to check for the actual implementation because the change set is to big.